MAVProxy/modules/mavproxy_map/__init__.py

```diff
@@ -187,7 +188,19 @@ def display_waypoints(self):
 
                     if (self.map_settings.loitercircle and
                         self.module('wp').wploader.wp_is_loiter(next_list[j])):
-                        self.mpstate.map.add_object(mp_slipmap.SlipCircle('Loiter Circle %u' % (next_list[j] + 1), 'LoiterCircles', polygons[i][j], abs(loiter_rad), (255, 255, 255), 2))
+                        wp = self.module('wp').wploader.wp(next_list[j])                    
+                        if wp.command != mavutil.mavlink.MAV_CMD_NAV_LOITER_TO_ALT and wp.param3 != 0:
```

Comment:
Annoyingly MAV_CMD_NAV_LOITER_TO_ALT has its radius parameter in param2 while the other loiter commands use param3...